POKEMON_OF_THE_DAY/.gitignore

```diff
@@ -1,2 +1,2 @@
-.env
```

Comment:
wait sorry my comment was unclear ... I think you should keep the `.env` in the gitignore